drafts/inlinemacros.md

```diff
@@ -80,39 +86,37 @@ by-value parameters and that are not side-effect free paths. Arguments to by-nam
 parameters are left alone.
```

Comment:
>Would our optimizer handle the influx of garbage temporaries?

It would. Actually optimizer itself introduces a huge amount of those during optimization and then cleans them up.
I'm supporting the idea to always introduce temporary values for call-by-value arguments.